v23.1/architecture/transaction-layer.md

```diff
@@ -117,6 +117,14 @@ Note that closed timestamps are valid even if the leaseholder changes, since the
 
 Closed timestamps provide the guarantees that are used to provide support for low-latency historical (stale) reads, also known as [Follower Reads](../follower-reads.html). Follower reads can be particularly useful in [multi-region deployments](../multiregion-overview.html).
 
+When [contention](../performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention) happens between transactions, the closed timestamp can provide some mitigation. Increasing the closed timestamp interval can reduce the likelihood of long-running transactions having their [timestamps pushed](#timestamp-cache), and therefore, reduces the likelihood of the transaction retrying. However, note that increasing the closed timestamp interval can have an impact on the following:
```

Comment:
This is correct, but a double-edged sword.  Considering a scenario in which a long-running transaction named txn1 holds write locks on some keys at time t=1, and txn2 is waiting to read those same keys at t=1:

1. If txn1 has it's timestamp pushed forward by the closed timestamp, moving its writes to t=2, txn2 may be able to proceed and read the keys at t=1.
2. If the closed timestamp interval is raised, that now means that long-running txn1 is causing txn2 to be long-waiting, as it can't read the keys at t=1 until txn1 is done or pushed into the future.

This is in addition to the other callouts listed below, and I think when talking about contention, it's worth mentioning that while increasing the closed timestamp interval may decrease retry errors, in may also increase lock wait latencies.